### PR TITLE
feat(shell-config): add neofetch message

### PR DESCRIPTION
**Pull request by madonuko**

### CODE REVIEW

File: ultramarine/shell-config/ultramarine-shell.sh [hunk 1]
```diff
@@ -61,6 +61,19 @@ alias yay=_aur_helper
 alias pacaur=_aur_helper
 alias paru=_aur_helper
 
+if ! [ -x "$(command -v neofetch)" ]; then
+    neofetch() {
+        echo 'neofetch is not installed.'
```
Comment (nothingneko) [inline ultramarine/shell-config/ultramarine-shell.sh]:
```suggestion
        echo 'Neofetch is not installed.'
```

File: ultramarine/shell-config/ultramarine-shell.sh [hunk 2]
```diff
@@ -61,6 +61,19 @@ alias yay=_aur_helper
...
+        echo 'While it is available via `sudo dnf in neofetch`, it does not come with the Ultramarine logo.'
```
Comment (nothingneko) [inline ultramarine/shell-config/ultramarine-shell.sh]:
```suggestion
        echo 'While neofetch is available in Ultramarine, it does not come with our logo, and is no longer maintained.'
```

File: ultramarine/shell-config/ultramarine-shell.sh [hunk 3]
```diff
@@ -61,6 +61,19 @@ alias yay=_aur_helper
...
+        echo 'We recommend fastfetch instead, which may be installed via:'
```
Comment (nothingneko) [inline ultramarine/shell-config/ultramarine-shell.sh]:
```suggestion
        echo 'We recommend fastfetch instead, which may be installed using:'
```

File: ultramarine/shell-config/ultramarine-shell.sh [hunk 4]
```diff
@@ -61,6 +61,19 @@ alias yay=_aur_helper
...
+        echo
+        echo '  sudo dnf in fastfetch'
```
Comment (nothingneko) [inline ultramarine/shell-config/ultramarine-shell.sh]:
```suggestion
        echo '  sudo dnf install fastfetch'
```
(for consistency with docs)

**Review by nothingneko — DISMISSED** [review]

File: ultramarine/shell-config/ultramarine-shell.sh [hunk 5]
```diff
@@ -61,6 +61,19 @@ alias yay=_aur_helper
...
+        echo
+        echo 'You may disable this message by installing hyfetch-neofetch or neofetch.'
```
Comment (lleyton) [inline ultramarine/shell-config/ultramarine-shell.sh]: neofetch is deprecated upstream

**Review by lleyton — CHANGES_REQUESTED** [review]

**Review by lleyton — APPROVED** [review]

**Pull request merged**